exts/omni.example.spawnPrims/tutorial/tutorial.md

```diff
@@ -1,171 +1,144 @@
 
-```Step 1.2.3:``` Enter a Project Folder Name and Extension Name.
-
-For this example we chose `kit-exts-spawnPrims` for the Project Folder Name and `omni.spawn.primitive` for our Extension Name.
-
-> 📝 **Note:** Choose the name of an extension differently for publishing. We suggest ```companyName.extdescrip1.descrip2``` 
+- For this example choose `kit-exts-spawnPrims` for the Project Folder Name and `omni.spawn.primitive` for the Extension Name.
 
 | ![spawnprim_tut_png5](images/spawnprim_tutorial5.png) | ![spawnprim_tut_png6](images/spawnprim_tutorial6.png) |
 | --- | --- |
 
+> 📝 **Note:** Choose the name of an extension differently for publishing. For Example: ```companyName.extdescrip1.descrip2``` 
 
-<br><br>
-
-> 📝 **Note:** To learn more about the Extension Manager head over to our documentations [here](https://docs.omniverse.nvidia.com/app_code/prod_extensions/ext_extension-manager.html?). 
-
-<br><br>
-
+After choosing the Extension Name two things will happen.
 
+- Visual Studio Code will start up displaying the template code
+- A new window will appear called "My Window" in Omniverse
 
-After choosing the Extension Name two things will happen.
-1. Visual Studio Code will start up displaying the template code
-2. A new window will appear called "My Window" in Omniverse
 
-| ![spawnprim_tut_png7](images/spawnprim_tutorial7.png) | ![spawnprim_tut_png8](images/spawnprim_tutorial8.png) |
-| ------ | ------ |
+    | ![spawnprim_tut_png7](images/spawnprim_tutorial7.png) | ![spawnprim_tut_png8](images/spawnprim_tutorial8.png) |
+    | ------ | ------ |
 
-Now that we created a new extension template, there will be a ```Click Me``` button. When clicked, a message in the Console Window in Omniverse Code recording that the button was clicked. We will use this button later to spawn a primitive.
+Now that you created a new extension template, there will be a ```Click Me``` button. When clicked, a message in the Console Window in Omniverse Code recording that the button was clicked. You will use this button later to spawn a primitive.
 
 Below is a visual to see the button in action.
 
-> 💡 **Tip:** If unsure where to locate the console, look to the bottom of Omniverse Code. There are tables listed *Content, NVIDIA Assets, Script Editor, etc*. Console should be listed there. If not try going to the top bar then select *Window>Utilities>Console*. Make sure there is a check mark next to Console and if not then select it for the window to show up.
-
+![spawnprim_tut_gif1](images/spawnprim_tutorial1.gif)
 
 
-![spawnprim_tut_gif1](images/spawnprim_tutorial1.gif)
+## Step 2: Updating the .toml File
 
-### Step 1.3: Updating the .toml File
+With the extension created you can also update how our extension will be viewed in the Extensions Tab. This is all done inside of the .toml file that comes with the template extension. Locate it inside of Visual Studio Code under the `config` folder. The file is called `extension.toml`.
```

Comment:
...you can update the metadata in the `extension.toml`. This is metadata used in the extension details of the Extensions Manager as well as to inform other systems of the Application. Locate the `extension.toml inside of Visual...